### PR TITLE
Fix TagsInput rendering with too much height

### PR DESCRIPTION
**Pull request by michsa**
Fixes smartprocure/bid-search#2151

The bid-search issue is caused by the `height: 100%` on TagsInput and its parent loader. Fine to remove (@stellarhoof confirmed) -- the only place that actually *needs* the `height: 100%` is the GreyVest search bar, which wraps TagsInput in a taller box. As long as we get it centered-looking via some alternative that doesn't mess with TagsInput's height elsewhere, we are good.

So, a couple of options for that:

1. Add styling to the wrapper box to v-center its contents (this is what I picked for this PR)
2. Remove the wrapper box and instead apply its styling to `.gv-search-bar .tags-input` in the CSS (this is more consistent with the other themes/demos)

Option 2 might be preferable in the long run, depending on what we decide to do with themes and loaders, and whether we eventually want to extend the search bar into a themeable component. However, it would still be broken under the current Loader implementation -- the loader wrapper around TagsQuery would continue to prevent it from growing with the grid (just as the wrapper box does now), resulting in a search box that's thinner than the button next to it.

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  This PR has failing tests. Please alleviate the errors and commit them

<details>
  <summary><a href="https://github.com/smartprocure/contexture-react/blob/feature/fix-tagsinput-height/s" target="_blank">s</a></summary>
  <code>  ● Storyshots › Search Components (Unthemed)|Example Types/ResultTable › Customizations

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|Example Types/ResultTable Customizations 1`

    - Snapshot
    + Received

    @@ -16,15 +16,12 @@
                  border-bottom: solid 2px #ccc
                }
              
        </style>
        <div
    -     style={
    -       Object {
    -         "height": "100%",
    -       }
    -     }
    +     className="loader"
    +     style={Object {}}
        >
          <table
            className="example-table"
          >
            <thead>

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|Example Types/ResultTable › Display Field Optional

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|Example Types/ResultTable Display Field Optional 1`

    - Snapshot
    + Received

    @@ -16,15 +16,12 @@
                  border-bottom: solid 2px #ccc
                }
              
        </style>
        <div
    -     style={
    -       Object {
    -         "height": "100%",
    -       }
    -     }
    +     className="loader"
    +     style={Object {}}
        >
          <table
            className="example-table"
          >
            <thead>

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|Example Types/Tags Query › Default

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|Example Types/Tags Query Default 1`

    - Snapshot
    + Received

    @@ -1,19 +1,12 @@
      <div
    -   style={
    -     Object {
    -       "height": "100%",
    -     }
    -   }
    +   className="loader"
    +   style={Object {}}
      >
        <div
          className="tags-input"
    -     style={
    -       Object {
    -         "height": "100%",
    -       }
    -     }
    +     style={Object {}}
        >
          <div
            style={
              Object {
                "alignItems": "center",

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|Example Types/Tags Query › Responsive

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|Example Types/Tags Query Responsive 1`

    - Snapshot
    + Received

    @@ -4,23 +4,16 @@
            "maxWidth": 500,
          }
        }
      >
        <div
    -     style={
    -       Object {
    -         "height": "100%",
    -       }
    -     }
    +     className="loader"
    +     style={Object {}}
        >
          <div
            className="tags-input"
    -       style={
    -         Object {
    -           "height": "100%",
    -         }
    -       }
    +       style={Object {}}
          >
            <div
              style={
                Object {
                  "alignItems": "center",

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|Example Types › Full Demo

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|Example Types Full Demo 1`

    - Snapshot
    + Received

    @@ -14,15 +14,12 @@
              "marginBottom": "25px",
            }
          }
        >
          <div
    -       style={
    -         Object {
    -           "height": "100%",
    -         }
    -       }
    +       className="loader"
    +       style={Object {}}
          >
            <input
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
    @@ -69,23 +66,16 @@
                    "marginBottom": "25px",
                  }
                }
              >
                <div
    -             style={
    -               Object {
    -                 "height": "100%",
    -               }
    -             }
    +             className="loader"
    +             style={Object {}}
                >
                  <div
                    className="tags-input"
    -               style={
    -                 Object {
    -                   "height": "100%",
    -                 }
    -               }
    +               style={Object {}}
                  >
                    <div
                      style={
                        Object {
                          "alignItems": "center",
    @@ -122,15 +112,12 @@
                    "marginBottom": "25px",
                  }
                }
              >
                <div
    -             style={
    -               Object {
    -                 "height": "100%",
    -               }
    -             }
    +             className="loader"
    +             style={Object {}}
                >
                  <input
                    node={
                      Object {
                        "field": "title",
    @@ -182,15 +169,12 @@
                    "marginBottom": "25px",
                  }
                }
              >
                <div
    -             style={
    -               Object {
    -                 "height": "100%",
    -               }
    -             }
    +             className="loader"
    +             style={Object {}}
                >
                  <div
                    className="contexture-facet"
                  >
                    <div
    @@ -420,15 +404,12 @@
                    "marginBottom": "25px",
                  }
                }
              >
                <div
    -             style={
    -               Object {
    -                 "height": "100%",
    -               }
    -             }
    +             className="loader"
    +             style={Object {}}
                >
                  <div
                    className="contexture-facet"
                  >
                    <div
    @@ -658,15 +639,12 @@
                    "marginBottom": "25px",
                  }
                }
              >
                <div
    -             style={
    -               Object {
    -                 "height": "100%",
    -               }
    -             }
    +             className="loader"
    +             style={Object {}}
                >
                  <div
                    className="contexture-number"
                  >
                    <div
    @@ -736,15 +714,12 @@
                    "marginBottom": "25px",
                  }
                }
              >
                <div
    -             style={
    -               Object {
    -                 "height": "100%",
    -               }
    -             }
    +             className="loader"
    +             style={Object {}}
                >
                  <div
                    className="contexture-number"
                  >
                    <div
    @@ -810,15 +785,12 @@
              </div>
              <div
                style={Object {}}
              >
                <div
    -             style={
    -               Object {
    -                 "height": "100%",
    -               }
    -             }
    +             className="loader"
    +             style={Object {}}
                >
                  <div
                    style={
                      Object {
                        "display": "flex",
    @@ -1011,15 +983,12 @@
                    "marginBottom": "25px",
                  }
                }
              >
                <div
    -             style={
    -               Object {
    -                 "height": "100%",
    -               }
    -             }
    +             className="loader"
    +             style={Object {}}
                >
                  <div
                    style={
                      Object {
                        "alignItems": "flex-end",
    @@ -1152,14 +1121,14 @@
                    "marginBottom": "25px",
                  }
                }
              >
                <div
    +             className="loader"
                  style={
                    Object {
                      "display": "inline-block",
    -                 "height": "100%",
                    }
                  }
                >
                  1
                </div>
    @@ -1175,15 +1144,12 @@
                      "justifyContent": "center",
                    }
                  }
                >
                  <div
    -               style={
    -                 Object {
    -                   "height": "100%",
    -                 }
    -               }
    +               className="loader"
    +               style={Object {}}
                  >
                    <table>
                      <thead>
                        <tr>
                          <th

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|Example Types › Geo filter & HERE maps

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|Example Types Geo filter & HERE maps 1`

    - Snapshot
    + Received

    @@ -22,15 +22,12 @@
                "flex": 1,
              }
            }
          >
            <div
    -         style={
    -           Object {
    -             "height": "100%",
    -           }
    -         }
    +         className="loader"
    +         style={Object {}}
            >
              <div
                style={
                  Object {
                    "display": "flex",

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|FilterAdder › Example

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|FilterAdder Example 1`

    - Snapshot
    + Received

    @@ -1,12 +1,9 @@
      <div>
        <div
    -     style={
    -       Object {
    -         "height": "100%",
    -       }
    -     }
    +     className="loader"
    +     style={Object {}}
        >
          <select
            onChange={[Function]}
          >
            <option

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|FilterAdder › With NestedPickerModal

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|FilterAdder With NestedPickerModal 1`

    - Snapshot
    + Received

    @@ -1,11 +1,8 @@
      <div
    -   style={
    -     Object {
    -       "height": "100%",
    -     }
    -   }
    +   className="loader"
    +   style={Object {}}
      >
        <div>
          <button
            onClick={[Function]}
          >

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|FilterList › Example

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|FilterList Example 1`

    - Snapshot
    + Received

    @@ -1,22 +1,16 @@
      <div
    -   style={
    -     Object {
    -       "height": "100%",
    -     }
    -   }
    +   className="loader"
    +   style={Object {}}
      >
        <div>
          <div
            className="filter-list-item"
          >
            <div
    -         style={
    -           Object {
    -             "height": "100%",
    -           }
    -         }
    +         className="loader"
    +         style={Object {}}
            >
              <div
                className="filter-field-label"
                onClick={[Function]}
                style={
    @@ -43,15 +37,12 @@
          </div>
          <div
            className="filter-list-item"
          >
            <div
    -         style={
    -           Object {
    -             "height": "100%",
    -           }
    -         }
    +         className="loader"
    +         style={Object {}}
            >
              <div
                className="filter-field-label"
                onClick={[Function]}
                style={
    @@ -78,15 +69,12 @@
          </div>
          <div
            className="filter-list-item"
          >
            <div
    -         style={
    -           Object {
    -             "height": "100%",
    -           }
    -         }
    +         className="loader"
    +         style={Object {}}
            >
              <div
                className="filter-field-label"
                onClick={[Function]}
                style={

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Grey Vest)|Example Types/Tags Query › Default

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Grey Vest)|Example Types/Tags Query Default 1`

    - Snapshot
    + Received

    @@ -535,23 +535,16 @@
              overflow-x: auto;
            }
          
        </style>
        <div
    -     style={
    -       Object {
    -         "height": "100%",
    -       }
    -     }
    +     className="loader"
    +     style={Object {}}
        >
          <div
            className="tags-input"
    -       style={
    -         Object {
    -           "height": "100%",
    -         }
    -       }
    +       style={Object {}}
          >
            <div
              style={
                Object {
                  "alignItems": "center",

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Grey Vest)|Example Types/Tags Query › Responsive

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Grey Vest)|Example Types/Tags Query Responsive 1`

    - Snapshot
    + Received

    @@ -542,23 +542,16 @@
              "maxWidth": 500,
            }
          }
        >
          <div
    -       style={
    -         Object {
    -           "height": "100%",
    -         }
    -       }
    +       className="loader"
    +       style={Object {}}
          >
            <div
              className="tags-input"
    -         style={
    -           Object {
    -             "height": "100%",
    -           }
    -         }
    +         style={Object {}}
            >
              <div
                style={
                  Object {
                    "alignItems": "center",

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Live Demos|IMDB › Quick Start

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Live Demos|IMDB Quick Start 1`

    - Snapshot
    + Received

    @@ -14,15 +14,12 @@
              "marginBottom": "25px",
            }
          }
        >
          <div
    -       style={
    -         Object {
    -           "height": "100%",
    -         }
    -       }
    +       className="loader"
    +       style={Object {}}
          >
            <input
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
    @@ -70,15 +67,12 @@
                <div>
                  <b>
                    MetaScore
                  </b>
                  <div
    -               style={
    -                 Object {
    -                   "height": "100%",
    -                 }
    -               }
    +               className="loader"
    +               style={Object {}}
                  >
                    <div
                      className="contexture-number"
                    >
                      <div
    @@ -153,15 +147,12 @@
                <div>
                  <b>
                    Genre
                  </b>
                  <div
    -               style={
    -                 Object {
    -                   "height": "100%",
    -                 }
    -               }
    +               className="loader"
    +               style={Object {}}
                  >
                    <div
                      className="contexture-facet"
                    >
                      <div
    @@ -274,15 +265,12 @@
                <div>
                  <b>
                    Actors
                  </b>
                  <div
    -               style={
    -                 Object {
    -                   "height": "100%",
    -                 }
    -               }
    +               className="loader"
    +               style={Object {}}
                  >
                    <div
                      className="contexture-facet"
                    >
                      <div
    @@ -390,15 +378,12 @@
                </div>
              </div>
            </div>
            <div>
              <div
    -           style={
    -             Object {
    -               "height": "100%",
    -             }
    -           }
    +           className="loader"
    +           style={Object {}}
              >
                <div
                  style={
                    Object {
                      "alignItems": "flex-end",
    @@ -416,27 +401,24 @@
                  }
                }
              >
                <h3>
                  <div
    +               className="loader"
                    style={
                      Object {
                        "display": "inline-block",
    -                   "height": "100%",
                      }
                    }
                  >
                    No Results
                  </div>
                </h3>
              </div>
              <div
    -           style={
    -             Object {
    -               "height": "100%",
    -             }
    -           }
    +           className="loader"
    +           style={Object {}}
              >
                <div
                  style={
                    Object {
                      "display": "flex",
    @@ -453,15 +435,12 @@
                    "justifyContent": "space-around",
                  }
                }
              >
                <div
    -             style={
    -               Object {
    -                 "height": "100%",
    -               }
    -             }
    +             className="loader"
    +             style={Object {}}
                />
              </div>
            </div>
          </div>
        </div>

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|QueryBuilder › One Filter

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|QueryBuilder One Filter 1`

    - Snapshot
    + Received

    @@ -115,15 +115,12 @@
                                    "width": "100%",
                                  }
                                }
                              >
                                <div
    -                             style={
    -                               Object {
    -                                 "height": "100%",
    -                               }
    -                             }
    +                             className="loader"
    +                             style={Object {}}
                                >
                                  <input
                                    onBlur={[Function]}
                                    onChange={[Function]}
                                    onFocus={[Function]}

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|QueryBuilder › One Filter with fields

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|QueryBuilder One Filter with fields 1`

    - Snapshot
    + Received

    @@ -144,15 +144,12 @@
                                    "width": "100%",
                                  }
                                }
                              >
                                <div
    -                             style={
    -                               Object {
    -                                 "height": "100%",
    -                               }
    -                             }
    +                             className="loader"
    +                             style={Object {}}
                                >
                                  <input
                                    onBlur={[Function]}
                                    onChange={[Function]}
                                    onFocus={[Function]}

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|QueryBuilder › One Filter with facet options

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|QueryBuilder One Filter with facet options 1`

    - Snapshot
    + Received

    @@ -115,15 +115,12 @@
                                    "width": "100%",
                                  }
                                }
                              >
                                <div
    -                             style={
    -                               Object {
    -                                 "height": "100%",
    -                               }
    -                             }
    +                             className="loader"
    +                             style={Object {}}
                                >
                                  <div
                                    className="contexture-facet"
                                  >
                                    <div

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|QueryBuilder › One Filter on a misplaced root

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|QueryBuilder One Filter on a misplaced root 1`

    - Snapshot
    + Received

    @@ -115,15 +115,12 @@
                                    "width": "100%",
                                  }
                                }
                              >
                                <div
    -                             style={
    -                               Object {
    -                                 "height": "100%",
    -                               }
    -                             }
    +                             className="loader"
    +                             style={Object {}}
                                >
                                  <input
                                    onBlur={[Function]}
                                    onChange={[Function]}
                                    onFocus={[Function]}

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)

  ● Storyshots › Search Components (Unthemed)|QueryBuilder › Multiple Filters

    expect(received).toMatchSnapshot()

    Snapshot name: `Storyshots Search Components (Unthemed)|QueryBuilder Multiple Filters 1`

    - Snapshot
    + Received

    @@ -132,15 +132,12 @@
                                    "width": "100%",
                                  }
                                }
                              >
                                <div
    -                             style={
    -                               Object {
    -                                 "height": "100%",
    -                               }
    -                             }
    +                             className="loader"
    +                             style={Object {}}
                                >
                                  <input
                                    onBlur={[Function]}
                                    onChange={[Function]}
                                    onFocus={[Function]}
    @@ -396,15 +393,12 @@
                                                "width": "100%",
                                              }
                                            }
                                          >
                                            <div
    -                                         style={
    -                                           Object {
    -                                             "height": "100%",
    -                                           }
    -                                         }
    +                                         className="loader"
    +                                         style={Object {}}
                                            >
                                              <input
                                                onBlur={[Function]}
                                                onChange={[Function]}
                                                onFocus={[Function]}
    @@ -599,15 +593,12 @@
                                                "width": "100%",
                                              }
                                            }
                                          >
                                            <div
    -                                         style={
    -                                           Object {
    -                                             "height": "100%",
    -                                           }
    -                                         }
    +                                         className="loader"
    +                                         style={Object {}}
                                            >
                                              <input
                                                onBlur={[Function]}
                                                onChange={[Function]}
                                                onFocus={[Function]}
    @@ -864,15 +855,12 @@
                                                            "width": "100%",
                                                          }
                                                        }
                                                      >
                                                        <div
    -                                                     style={
    -                                                       Object {
    -                                                         "height": "100%",
    -                                                       }
    -                                                     }
    +                                                     className="loader"
    +                                                     style={Object {}}
                                                        >
                                                          <div
                                                            className="contexture-facet"
                                                          >
                                                            <div
    @@ -1150,15 +1138,12 @@
                                                            "width": "100%",
                                                          }
                                                        }
                                                      >
                                                        <div
    -                                                     style={
    -                                                       Object {
    -                                                         "height": "100%",
    -                                                       }
    -                                                     }
    +                                                     className="loader"
    +                                                     style={Object {}}
                                                        >
                                                          <input
                                                            onBlur={[Function]}
                                                            onChange={[Function]}
                                                            onFocus={[Function]}
    @@ -1365,15 +1350,12 @@
                                    "width": "100%",
                                  }
                                }
                              >
                                <div
    -                             style={
    -                               Object {
    -                                 "height": "100%",
    -                               }
    -                             }
    +                             className="loader"
    +                             style={Object {}}
                                >
                                  <input
                                    onBlur={[Function]}
                                    onChange={[Function]}
                                    onFocus={[Function]}
    @@ -1649,15 +1631,12 @@
                                                "width": "100%",
                                              }
                                            }
                                          >
                                            <div
    -                                         style={
    -                                           Object {
    -                                             "height": "100%",
    -                                           }
    -                                         }
    +                                         className="loader"
    +                                         style={Object {}}
                                            >
                                              <div
                                                className="contexture-number"
                                              >
                                                <div
    @@ -1893,15 +1872,12 @@
                                                "width": "100%",
                                              }
                                            }
                                          >
                                            <div
    -                                         style={
    -                                           Object {
    -                                             "height": "100%",
    -                                           }
    -                                         }
    +                                         className="loader"
    +                                         style={Object {}}
                                            >
                                              <input
                                                onBlur={[Function]}
                                                onChange={[Function]}
                                                onFocus={[Function]}
    @@ -2239,15 +2215,12 @@
                                                            "width": "100%",
                                                          }
                                                        }
                                                      >
                                                        <div
    -                                                     style={
    -                                                       Object {
    -                                                         "height": "100%",
    -                                                       }
    -                                                     }
    +                                                     className="loader"
    +                                                     style={Object {}}
                                                        >
                                                          <input
                                                            onBlur={[Function]}
                                                            onChange={[Function]}
                                                            onFocus={[Function]}
    @@ -2443,15 +2416,12 @@
                                                            "width": "100%",
                                                          }
                                                        }
                                                      >
                                                        <div
    -                                                     style={
    -                                                       Object {
    -                                                         "height": "100%",
    -                                                       }
    -                                                     }
    +                                                     className="loader"
    +                                                     style={Object {}}
                                                        >
                                                          <input
                                                            node={
                                                              Object {
                                                                "context": null,
    @@ -2691,15 +2661,12 @@
                                                "width": "100%",
                                              }
                                            }
                                          >
                                            <div
    -                                         style={
    -                                           Object {
    -                                             "height": "100%",
    -                                           }
    -                                         }
    +                                         className="loader"
    +                                         style={Object {}}
                                            >
                                              <input
                                                onBlur={[Function]}
                                                onChange={[Function]}
                                                onFocus={[Function]}

      at match (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:20)
      at node_modules/@storybook/addon-storyshots/dist/test-bodies.js:39:10
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/api/snapshotsTestsTemplate.js:35:33)
</code>
</details>
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>We were able to automatically fix some formatting issues in this PR for you!</td>
    </tr>
  </tbody>
</table>


### Some things that were possibly fixed:

- Code that could be fixed via the --fix flag
- Formatting that could be fixed by prettier

Take a look at this commit to see what happened in detail: f9ee5b1cb52e1ebed6f4f85a2f4c1fad60872aff


And look at this wiki page to see the reasoning behind the ESLint rules: https://github.com/smartprocure/eslint-config-smartprocure/wiki/Rules-and-Why-We-Chose-Them
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 46f5d1f8492875b7a1dc7325c87250d1f6243999
</p>

File: src/utils/StripedLoader.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { loading } from '../styles/generic'
 
 let StripedLoader = (Component, style = {}) =>
   observer(({ isLoading, ...props }) => (
-    <div style={{ height: '100%', ...style, ...(isLoading && loading) }}>
+    <div style={{ ...style, ...(isLoading && loading) }} className="loader">
```
Comment (stellarhoof) [inline src/utils/StripedLoader.js]: What's the classname for?

Comment (michsa) [reply src/utils/StripedLoader.js]: I added it to identify the loaders better when inspecting, and figured it was potentially useful enough to leave in there. It's not actually *doing* anything though, so it's fine to take out if you want to.

File: src/themes/greyVest/index.js [hunk 2]
```diff
@@ -350,7 +350,11 @@ export let GVStyle = () => (
         /*background: #f6f6f6;*/
       }
       .gv-search-bar .gv-box {
-        padding: 0;
+        padding: 8px 10px;
```
Comment (michsa) [inline src/themes/greyVest/index.js]: Hope this is alright -- it was bothering me to see the search input so close to the left with all that vertical space, so I snuck in a bit of padding. It still looks nice with a few rows of entered tags (I checked) 🙂

Comment (stellarhoof) [reply src/themes/greyVest/index.js]: If it looks nicer then better :)

File: src/utils/StripedLoader.js (hunk 1 above)
Comment (stellarhoof) [reply src/utils/StripedLoader.js]: Yeah let's remove anything unused.

**Comment by daedalus28** [comment]
This looks good to go - just fix the conflicts (version and also no more snapshots!)

**Pull request merged**